contract/rust/build.js

```diff
@@ -0,0 +1,8 @@
+const shell = require('shelljs');
+
+shell.fatal = true; // same as "set -e"
+
+shell.cd('contract');
+// Note: see flags in ./cargo/config
```

Comment:
@mikedotexe feel like specifying all flags here would be less confusing

Reply:
This is very normal in Rust, right? And if people want to run a debug version they have less to copy/paste into their terminal, yes? Ideally we'd also have the `target = "wasm32-unknown-unknown"` in the `.cargo/config` but that breaks tests because Rust hasn't addressed that issue yet.

Reply:
If people want to run the release build, then `yarn build:contract` is a good shortcut, but if folks want to run a debug version or other typical commands starting with `cargo…` then we're asking them to prepend an ENV variable to it? That's weird and ugly, imo. And windows will be `set RUSTFLAGs=…` while 'nix and OS X will be `RUSTFLAGS=…`
I don't see the problem here, it is conventional for the language and makes `cargo` commands shorter.

Reply:
Actually, it's a little worse than I thought, they have to set an env variable that persists in the command prompt session. This approach is per project and I think it's a better fit